dev/com.ibm.ws.microprofile.openapi/resources/META-INF/services/org.eclipse.microprofile.openapi.spi.OASFactoryResolver

```diff
@@ -0,0 +1 @@
+com.ibm.ws.microprofile.openapi.OASFactoryResolverImpl
```

Comment:
I guess we don't have a way to register our `OASFactoryResolverImpl` using OSGi?